app/scripts/models/account.js

```diff
@@ -213,16 +215,28 @@ define([
         self.set(updatedSessionData);
 
         if (! self.get('verified')) {
-          return self._fxaClient.signUpResend(relier, self.get('sessionToken'));
+          return self._fxaClient.signUpResend(
+            relier,
+            self.get('sessionToken'),
+            {
+              resume: options.resume
+            }
```

Comment:
I assume the "options" argument here is only for options to pass on to the fxaClient, so I'm wondering whether it mskes sense to destructure it and pick out just this one field, or whether we should just pass it on directly.  No strong preference either way, just something that stood out to me.